numba/tests/test_array_reductions.py

```diff
@@ -352,6 +352,7 @@ def variations(a):
             yield a
             a[a % 4 >= 1] = 3.5
             yield a
+            yield np.array([]) #Empty array case
```

Comment:
Seems like the placement of this test case is incorrect. This case should ideally be present as an empty test case within the `check_median_basic`.

Reply:
Thank you for reviewing my PR and highlighting the issue. I appreciate your feedback and will address it as follows:

-Move the changes to the correct function.
-Write and include a proper release note.

Thank you for your patience!